src/Assets/Styles/site.scss

```diff
@@ -75,6 +75,10 @@ $govuk-global-styles: true;
 }
 
 .search-results-header {
+  @include govuk-clearfix;
+
+  border-bottom: 1px solid lightgrey;
```

Comment:
Please update `lightgrey` to `$govuk-border-colour` ;-)

Reply:
Yes we can 👍